<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tabz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QTextBrowser" name="transactionBrowser">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>41</y>
      <width>250</width>
      <height>430</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>471</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="tabNameLabel">
       <property name="text">
        <string>Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tabName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="disableFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="purchaseButton">
        <property name="text">
         <string>Add Purchase</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="transferButton">
        <property name="text">
         <string>Add Transfer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="balanceButton">
        <property name="text">
         <string>Balance</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="tabActionLabel">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tab Actions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="memberLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Members</string>
    </property>
   </widget>
   <widget class="QLabel" name="reqLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Required Transactions</string>
    </property>
   </widget>
   <widget class="QListWidget" name="reqList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>360</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="payButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pay selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="payAllButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pay all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="memList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="updateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="trCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Transactions</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="opCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Operations</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="autoCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Automatic update</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>101</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="addMemButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add member</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Exclude member</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeMemButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove member</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="childLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SubTabs</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="openChildButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeChildButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="returnButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Return to main tab</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="childList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="childCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Include SubTabs</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionOpen_Tab"/>
    <addaction name="actionSave_Tab"/>
    <addaction name="actionReplicate_Tab"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTab">
    <property name="title">
     <string>Tab</string>
    </property>
    <addaction name="actionAdd_member"/>
    <addaction name="actionRemove_member"/>
    <addaction name="actionDirectory"/>
    <addaction name="actionAutosave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTab"/>
  </widget>
  <action name="actionOpen_Tab">
   <property name="text">
    <string>Open Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_Tab">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionReplicate_Tab">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Replicate Tab</string>
   </property>
  </action>
  <action name="actionAdd_member">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add member</string>
   </property>
  </action>
  <action name="actionRemove_member">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove member</string>
   </property>
  </action>
  <action name="actionAutosave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Autosave</string>
   </property>
  </action>
  <action name="actionDirectory">
   <property name="text">
    <string>Open directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
